src/include/pbs_nodes.h

```diff
@@ -265,7 +266,7 @@ struct	pbsnode {
 	long			 nd_nsnfree;	/* number of VPs free */
 	long			 nd_ncpus;	/* number of phy cpus on node */
 	short			 nd_written;	/* written to nodes file */
-	unsigned short		 nd_state;	/* state of node */
+	unsigned long		 nd_state;	/* state of node */
```

Comment:
No issues, I am fine with long. 

https://www.postgresql.org/docs/9.3/static/datatype-numeric.html 
https://www.postgresql.org/docs/9.3/static/ecpg-variables.
So it should be able to hold a fairly large value, until we cross the 4 byte limit. 

For now, you are not crossing that limit, so the saves will be fine. Or you could make the value an int for now. If we cross this later, we will have to update the schema and that would need to take care of upgrades.

Summary: Keep it long or int, your choice. If we cross and start failing to save in DB, any basic test will catch that and we can fix that at that point in the database.